### ISSUE TITLE
MAC OS X: Issue building the json target

### ISSUE THREAD

**Issue by Guillaumedrlz** [ARDroneSDK3/ARSDKBuildUtils#7]
Hello,
I'm facing errors when building the SDK on MAC OS X 10.9.5. The json target seems to be the cause and this part of the log shows the error:

```
Entering </Users/gd/Documents/ParrotSDK/ARSDKBuildUtils/Targets/Unix/Build/libjson>
Running <make -j 8 install>
 cd /Users/gd/Documents/ParrotSDK/ARSDKBuildUtils/../json-c/json-c-0.9 && automake-1.14 --gnu 
automake-1.14: warning: autoconf input should be named 'configure.ac', not 'configure.in'
configure.in:6: warning: AM_INIT_AUTOMAKE: two- and three-arguments forms are deprecated.  For more info, see:
configure.in:6: http://www.gnu.org/software/automake/manual/automake.html#Modernize-AM_005fINIT_005fAUTOMAKE-invocation
configure.in:14: error: required file './compile' not found
configure.in:14:   'automake --add-missing' can install 'compile'
automake-1.14: warning: autoconf input should be named 'configure.ac', not 'configure.in'
Makefile.am:5: error: Libtool library used but 'LIBTOOL' is undefined
Makefile.am:5:   The usual way to define 'LIBTOOL' is to add 'LT_INIT'
Makefile.am:5:   to 'configure.in' and run 'aclocal' and 'autoconf' again.
Makefile.am:5:   If 'LT_INIT' is in 'configure.in', make sure
Makefile.am:5:   its definition is in aclocal's search path.
make: *** [/Users/gd/Documents/ParrotSDK/ARSDKBuildUtils/../json-c/json-c-0.9/Makefile.in] Error 1
Error while running <make -j 8 install>
Failed to build libjson
Returning to </Users/gd/Documents/ParrotSDK/ARSDKBuildUtils>
Error while running Common_BuildConfigureLibrary with args:
 -- confdirSuffix -> 
 -- target -> Unix
 -- lib -> json
 -- noSharedObjects -> False
 -- extraArgs -> ['CFLAGS="-fPIC"']
 -- installSubDir -> 
 -- isLib -> True
 -- inhouse -> False
 -- clean -> False
 -- debug -> False
 -- stripVersionNumber -> False
Error while running Unix_BuildLibrary with args:
 -- nodeps -> False
 -- target -> Unix
 -- lib -> json
 -- inhouse -> False
 -- clean -> False
 -- debug -> False
```

Am I missing something? Let me know if you want the full log.
Thanks

**Comment by Guillaumedrlz** [comment]
It seems I solved my problem. Here are the steps :
1. I used brew to install the tools that were missing: wget, automake, autoconf, libtool, yasm and nasm. libtool will get installed as glibtool to not overwrite the native Mac version. zlib1g-dev wasn't available through brew but comes natively with Mac so I ignored it.
2. The curl target wasn't building at this point so I installed xCode. It solved the issue for curl but I was now stuck at the json target.
3. I then did a ```./SDK3Build.py --all-cleanup``` and deleted the folder in which I cloned the git.
4. I rebooted my computer and started from the beginning again by cloning the git and launching the build. This time, it went fine and all targets were built.

My guess is that installing xCode made the trick.
